includes/plugin-update-checker/Puc/v4p11/Plugin/Info.php

```diff
@@ -77,16 +77,23 @@ protected function validateMetadata($apiResponse) {
 				if ($apiResponse->status === 'expired') {
 					return new WP_Error(
 						'puc-expired-license', 
-						'The plugin license is expired.'
+						'The plugin cannot be updated because it is expired.'
+					);	
+				}
+
+				if ($apiResponse->status === 'inactive') {
+					return new WP_Error(
+						'puc-inactive-license', 
+						'The plugin cannot be updated because it is inactive.'
 					);	
 				}
 
 				return new WP_Error(
-					'puc-invalid-metadata',
-					"The plugin metadata file does not contain the required 'name' and/or 'version' keys."
+					'puc-invalid-or-expired',
+					"The plugin might be invalid or expired."
```

Comment:
```suggestion
					"The plugin license is invalid or expired."
```

I'd change the wording here: "might be" -> "is".